# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1325856551 18000
#      Fri Jan 06 08:29:11 2012 -0500
# Branch stable
# Node ID 933bf1b4ab29afd7a860583e25374500771200f2
# Parent  e8bad0dc263c95d09dbb6a0a2a6b68cfec67efee
* lo-sysdep.cc (octave_chdir): Use gnulib::chdir.

diff --git a/liboctave/lo-sysdep.cc b/liboctave/lo-sysdep.cc
--- a/liboctave/lo-sysdep.cc
+++ b/liboctave/lo-sysdep.cc
@@ -72,17 +72,17 @@ octave_chdir (const std::string& path_ar
 {
   std::string path = file_ops::tilde_expand (path_arg);
 
 #if defined (__WIN32__) && ! defined (__CYGWIN__)
   if (path.length() == 2 && path[1] == ':')
     path += "\\";
 #endif
 
-  return chdir (path.c_str ());
+  return gnulib::chdir (path.c_str ());
 }
 
 #if defined (__WIN32__) && ! defined (__CYGWIN__)
 
 pid_t
 octave_popen2 (const std::string& cmd, const string_vector& args, bool sync_mode,
     int *fildes, std::string& msg)
 {
